setup.py

```diff
@@ -43,7 +43,7 @@
     license=about["__license__"],
     packages=find_packages(include=["aws_codeseeder", "aws_codeseeder.*"]),
     keywords=["aws", "cdk"],
-    python_requires=">=3.7, <3.12",
+    python_requires=">=3.7, <3.13",
```

Comment:
We should update the `classifiers` property as well. Currently it only includes Python version 3.7 to 3.9.